packages/legend-studio/src/components/editor/edit-panel/uml-editor/AssociationEditor.tsx

```diff
@@ -161,6 +181,7 @@ const AssociationPropertyBasicEditor = observer(
       <div className="property-basic-editor">
         <input
           className="property-basic-editor__name"
+          id={property.uuid}
```

Comment:
why do we need this

Reply:
We want only the input attribute with duplicate property to go red so using `id` to differentiate that

Reply:
Won't be needing with updated logic